src/Haskell/Ide/Engine/Plugin/Brittany.hs

```diff
@@ -25,47 +25,98 @@ data FormatParams = FormatParams Int Uri (Maybe Range)
 
 brittanyDescriptor :: PluginId -> PluginDescriptor
 brittanyDescriptor plId = PluginDescriptor
-  { pluginId       = plId
-  , pluginName     = "Brittany"
-  , pluginDesc     = "Brittany is a tool to format source code."
-  , pluginCommands = []
+  { pluginId                 = plId
+  , pluginName               = "Brittany"
+  , pluginDesc               = "Brittany is a tool to format source code."
+  , pluginCommands           = [ PluginCommand "formatText"
```

Comment:
I would prefer if we can use the `pluginFormattingProvider`s directly rather than calling a `PluginCommand`, since `PluginCommands` get exposed to LSP

Reply:
I think what we really need is some kind of way to let plugins call each other. A bunch of plugins need to format after they do something.
Maybe a separate issue for discussion would be nice?